mappings/net/minecraft/entity/ai/AiGoalJumpOutOfLiquid.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS vu net/minecraft/entity/ai/AiGoalJumpOutOfLiquid
```

Comment:
I think that makes sense. In the logic the only thing it tries to do is jump so I figured it was trying to get out out the liquid. It's basically the same action as swimming though.